modules/coverage-page-overview.adoc

```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// * operating/managing-compliance.adoc
+
+:_mod-docs-content-type: CONCEPT
+[id="coverage-page-overview_{context}"]
+= Coverage page overview
+
+The *Checks* view lists the profile checks and enables you to easily navigate and understand your compliance status.
+
+The profile check information is organized in the following groups:
+
+* *Check*: The name of the profile check.
+* *Controls*: Shows the various controls associated with each check.
+* *Fail status*: Shows the checks that have failed and require your attention.
+* *Pass status*: Shows the checks that have been successfully passed.
+* *Other status*: Shows the checks with a status other than pass or fail, such as warnings or informational statuses.
```

Comment:
"Manual status" should probably have its own bullet now. The manual status count was pulled out from the "Other status" column.

```suggestion
* *Manual status*: Shows the checks that require a manual review due to the need for additional organizational or technical knowledge that is not automatable.
* *Other status*: Shows the checks with a status other than pass or fail, such as warnings or informational statuses.
```

Same with the clusters view right below here.